index.html

```diff
@@ -197,5 +197,7 @@ <h2 class="m-0 gray">
     </footer>
```

Comment:
[OPTIONAL] The mobile version of the main page does not have the footer, it ends at the `more` button. Kindly check and fix it in order to conform with the design

 Your design            |  The figma design 
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-24 08-35-20](https://user-images.githubusercontent.com/65233860/203733586-9057610c-1f9e-4009-9a97-ea2b644ea0cd.png)  |  ![Screenshot from 2022-11-24 08-35-09](https://user-images.githubusercontent.com/65233860/203733603-0a731da5-eb7b-4d30-9d67-7d33cf2c033b.png)